scripts/start

```diff
@@ -16,6 +16,17 @@ if [[ $UID != 0 ]]; then
     exit 1
 fi
 
+[[ -f /etc/lsb-release ]] && source /etc/lsb-release
+
+if [ "${DISTRIB_ID}"="Ubuntu" ] && [ "${DISTRIB_RELEASE}"="20.04" ]; then
```

Comment:
I think checking for a version less than 22.04 may be ideal rather than checking for a specific version. I am guessing the version you used as the check is the LTS release?

Reply:
Yes, other versions are wayless common, so I didn't want to implement a complicated semver check